SYCL/Regression/set-arg-local-accessor.cpp

```diff
@@ -0,0 +1,79 @@
+// REQUIRES: opencl
+//
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -lOpenCL -o %t.out
+// RUN: env SYCL_DEVICE_FILTER=opencl %CPU_RUN_PLACEHOLDER %t.out
+// RUN: env SYCL_DEVICE_FILTER=opencl %GPU_RUN_PLACEHOLDER %t.out
+// RUN: env SYCL_DEVICE_FILTER=opencl %ACC_RUN_PLACEHOLDER %t.out
+
+#include <sycl/sycl.hpp>
+
+using namespace sycl;
+
+const char *clSource = R"(
+    kernel void test(global int *a, local float *b, int n) {
+        if (get_local_id(0) != 0)
+            return;
+        for (int i = 0; i < n; i++)
+            b[i] = i;
+        barrier(CLK_LOCAL_MEM_FENCE);
+
+        bool ok = true;
+        for (int i = 0; i < n; i++)
+            ok &= (b[i] == i);
+
+        a[get_global_id(0)] = ok;
+    }
+)";
+
+void handle(cl_int status) {
+  if (status != CL_SUCCESS)
+    throw std::runtime_error("OpenCL error occurred");
+}
+
+kernel getKernel(const queue &Q) {
+  cl_int status;
```

Comment:
```suggestion
  cl_int status = CL_SUCCESS;
```

Reply:
Done in 8cfb6f4c98e7f7d71f6d7f2da3d8bb37e0496759